src/Templates/src/templates/maui-resourcedictionary-xaml/NewContent1.xaml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ResourceDictionary xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="MauiApp1.NewContent1">
```

Comment:
Maybe could be nice to call it **Resource1** to have the same experience as before with Resource Dictionaries in other frameworks 
![image](https://user-images.githubusercontent.com/6755973/156756176-b571ac9f-1f45-4d8d-9320-fb08a2d60d1f.png)

Reply:
You mean Resource1 or Dictionary1?

Reply:
Dictionary1